tests/functional/test_s3.py

```diff
@@ -462,19 +461,35 @@ def test_s3_copy_object_with_empty_response(self):
         self.assertEqual(response["ResponseMetadata"]["HTTPStatusCode"], 200)
         self.assertTrue("CopyObjectResult" in response)
 
-    def test_s3_copy_object_with_incomplete_response(self):
+    def test_s3_copy_object_with_200_error_response(self):
         self.client, self.http_stubber = self.create_stubbed_s3_client(
             region_name="us-east-1"
         )
-
-        incomplete_body = b'<?xml version="1.0" encoding="UTF-8"?>\n\n\n'
-        self.http_stubber.add_response(status=200, body=incomplete_body)
-        with self.assertRaises(ResponseParserError):
+        error_body = (
+            b"<Error>"
+            b"<Code>SlowDown</Code>"
+            b"<Message>Please reduce your request rate.</Message>"
+            b"</Error>"
+        )
+        self.http_stubber.add_response(status=200, body=error_body)
+        self.http_stubber.add_response(status=200, body=error_body)
+        self.http_stubber.add_response(status=200, body=error_body)
+        self.http_stubber.add_response(status=200, body=error_body)
+        self.http_stubber.add_response(status=200, body=error_body)
```

Comment:
why do we have to add the response 5 times?

Reply:
This was to ensure we are reaching the 4 max retries to assert we are raising the `ClientError`.

Reply:
Ah, my bad. I missed that on the first read. Thank you!